modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -730,8 +731,18 @@ export const console: ConsoleNS = {
                                             }
                                         }
                                     },
-                                    mappedAtributeHint: "Instead of sending the default attribute, the custom " +
-                                        "mapped attribute will be sent",
+                                    mappedAtributeHint: "Enter the custom attribute that should be requested " + 
+                                        "instead of the default attribute.",
+                                    mappingRevert: {
+                                        confirmPrimaryAction: "Confirm",
+                                        confirmSecondaryAction: "Cancel",
+                                        confirmationContent: "This custom attributes mapped to relevant default " +
+                                            "attributes will be changed back to the default mapped values by this " +
+                                            "action. Please use with caution since the you will not be able to " +
+                                            "retrieve the mapped custom attribute values.",
```

Comment:
```suggestion
                                        confirmationContent: "The mapped custom attributes will change  " +
                                            "back to the default attribute values. " +
                                            "Please proceed with caution since you will not be able to " +
                                            "retrieve the mapped custom attribute values.",
```